vignettes/v022_example.Rmd

```diff
@@ -0,0 +1,348 @@
+---
+title: "Demonstrating package & data setup and handling of sf objects: a case with read_soilmap()"
+author: "Floris Vanderhaeghe"
+date: "2020-03-30"
+bibliography: references.bib
+output: 
+  rmarkdown::html_vignette:
+    pandoc_args:
+        - --csl
+        - research-institute-for-nature-and-forest.csl
+vignette: >
+  %\VignetteIndexEntry{022. Demonstrating package & data setup and handling of sf objects: a case with read_soilmap()}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+options(stringsAsFactors = FALSE)
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = FALSE
+)
+```
+
+_Note: for more information on data storage and locations, see `vignette("v020_datastorage")`._
+_The below workflow is an application and demonstrates current setup from R (output is mostly not included)._
+
+## Minimal setup for using `read_soilmap()`
+
+```{r}
+library(remotes)
+```
+
+### Install and load `n2khab`
+
+```{r}
+install_github("inbo/n2khab",
+               build_vignettes = TRUE,
+               upgrade = TRUE)
+```
+
+```{r eval=TRUE}
+library(n2khab)
+```
+
+### Create the `n2khab_data` directory with 2 subdirectories (in the working directory)
+
+```{r}
+n2khab_data_path <- fileman_folders()
+```
+
+You can also provide a custom root 'path' to `fileman_folders()`, but then take a location that is a (grand-grand-...) parent of the current working directory.
+This is because reading functions of `n2khab` will by default use the first `n2khab_data` directory they can find, starting from the working directory and sequentially climbing up one directory level at a time.
+
+### Download the `soilmap_simple` data source in the correct place
+
+The `soilmap_simple` data source is a GeoPackage, provided and documented at
+[Zenodo](https://doi.org/10.5281/zenodo.3732903).
+It belongs to the collection of processed data ([Zenodo-link](https://zenodo.org/communities/n2khab-data-processed)).
+
+```{r}
+soilmap_simple_path <- file.path(n2khab_data_path, "20_processed/soilmap_simple")
+dir.create(soilmap_simple_path)
+download_zenodo(doi = "10.5281/zenodo.3732903",
```

Comment:
:+1:  Well, you're the third one (I know of) to mention this in two days time :3rd_place_medal:. You're more than welcome to join at inbo/inborutils#79 and share your output for the [proposed code](https://github.com/inbo/inborutils/issues/79#issuecomment-607860187) with `curl_download()`; I expect it will be a problem that happens on Windows machines (I couldn't reproduce it in Linux) that needs to be solved at some level, perhaps not in `download_zenodo()` but we'll see. Earlier before, it was not a problem.

Fortunately it is just a message problem; downloads still do happen.